aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -213,13 +213,20 @@
    * Enables eTag checks for S3 PUT and MULTIPART upload requests.
    */
   public static final String S3_CHECKSUM_ENABLED = "s3.checksum-enabled";
+
+  /**
+   * Sets the size for batch deletion.
+   */
```

Comment:
I think we also need a config for this

Reply:
yes definitley, added!

Reply:
Nice finding, I would even consider put a comment to reference where is 250 comes from https://github.com/apache/hadoop/commit/56dee667707926f3796c7757be1a133a362f05c9 until benchmark is completed